buildout.cfg

```diff
@@ -37,7 +37,7 @@ zope.testrunner = 4.4.3
 [script]
 recipe = zc.recipe.egg
 dependent-scripts = true
-interpreter = python
```

Comment:
Is this really needed for 3.4 support? Looks like an unrelated change. I'm personally used to having bin/python in all my buildouts. Is there a good reason to do it otherwise?s